cilium/cmd/bpf_ct_list.go

```diff
@@ -57,6 +63,36 @@ func getMaps(eID string) []*ctmap.Map {
 	return ctmap.LocalMaps(&dummyEndpoint{ID: id}, true, true)
 }
 
+func doDumpEntries(m ctmap.CtMap) {
+	var out string
+	var err error
+
+	if timeDiff {
+		params := daemon.NewGetHealthzParamsWithTimeout(5 * time.Second)
```

Comment:
Usually the `cilium bpf` commands don't require connection to the agent. Can't the developer pass a flag with the clock type as a fallback?

Reply:
I thought about this option, but I decided that it would not be very ergonomic to do so and opted to make their (and our) lives easier. Especially because users would have to use the same first-level command (`cilium`) to do this:
```
cilium status --verbose
cilium bpf ct list --time-diff --clock-source xxx
```

I believe connecting the agent _only_ if a specific flag is give is a good tradeoff because the tool will continue to work on its basic form (as before) without depending on the agent.

Reply:
> I thought about this option, but I decided that it would not be very ergonomic to do so and opted to make their (and our) lives easier. Especially because users would have to use the same first-level command (`cilium`) to do this:
> 
> ```
> cilium status --verbose
> cilium bpf ct list --time-diff --clock-source xxx
> ```
> 
> I believe connecting the agent _only_ if a specific flag is give is a good tradeoff because the tool will continue to work on its basic form (as before) without depending on the agent.

What I'm suggesting is to have a flag that allows someone to pass the `time-diff` as a value of its own, this is useful even if the agent is not available. I'm not saying to replace the `params := daemon.NewGetHealthzParamsWithTimeout(5 * time.Second)` with the flag.

Reply:
I see, so

```
cilium bpf ct list --time-diff
```

contacts the server, and:


```
cilium bpf ct list --time-diff --time-diff-clocksource=jiffies|ktime
```

does not contact the server.

Makes sense!

Reply:
Updated version includes this change.